server/src/migration/1555336867562-AskedQuestion.ts

```diff
@@ -0,0 +1,13 @@
+import {MigrationInterface, QueryRunner} from 'typeorm';
+
+export class AskedQuestion1555336867562 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "generated_question" RENAME TO "question"`);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "generated_answer" RENAME TO "answer"`);
```

Comment:
should be in up() method
add to down() backward renaming:
`
await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "answer" RENAME TO "generated_answer"`);
await queryRunner.query(`ALTER TABLE "asked_question" ALTER COLUMN "question" RENAME TO "generated_question"`);
`